### PR TITLE
Commentary section

### PR DESCRIPTION
**Pull request by fedeGiussa25**
## Summary

- Added commentary section

## Screenshots

[Change!] Show the screenshots of the views you modified.

#### Galaxy S (or similar)
Peek 2020-07-28 13-04

#### Pixel 2 (or similar)
Peek 2020-07-28 13-06

#### Pixel 3 or 3XL (or similar)
Peek 2020-07-28 13-09

### iOS
- Not for this training.

## Trello Card

https://trello.com/c/zUfuFelF/16-n-33-tabla-de-comentarios

### CODE REVIEW

File: src/app/screens/Detail/components/CommentarySection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  const [showAll, setShowAll] = useState(false);
+  const list = showAll ? data : data.slice(0, 2);
+
+  return (
+    <View style={styles.container}>
+      {list.map(item => (
+        <Comment key={item.id} item={item} />
+      ))}
+      {!showAll && (
+        <TouchableOpacity onPress={() => setShowAll(true)} style={styles.button}>
```
Comment (mattgle) [inline src/app/screens/Detail/components/CommentarySection/index.tsx:19]: extract this arrow function

File: src/app/screens/Detail/components/CommentarySection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const CommentarySection = ({ data }: CommentListProps) => {
+  const [showAll, setShowAll] = useState(false);
+  const list = showAll ? data : data.slice(0, 2);
+
+  return (
+    <View style={styles.container}>
+      {list.map(item => (
+        <Comment key={item.id} item={item} />
+      ))}
```
Comment (mattgle) [inline src/app/screens/Detail/components/CommentarySection/index.tsx:17]: I think you can fix the format here and you can merge these 3 lines into 1.

_A comment, in this case this function is taking just 3 lines and that's why is okay but if this functions gets bigger you should extract it._

Comment (mattgle) [reply src/app/screens/Detail/components/CommentarySection/index.tsx:17]:
```suggestion
      {list.map(item => <Comment key={item.id} item={item} />)}
```

Comment (fedeGiussa25) [reply src/app/screens/Detail/components/CommentarySection/index.tsx:17]: The linter is auto-fixing this. I can't change it

File: src/app/screens/Detail/components/DataSection/index.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import imagePlaceholder from '@assets/img_book6.png';
+import { BookProps } from '@interfaces/books';
+import { useRoute } from '@react-navigation/native';
```
Comment (sfernandez11) [inline src/app/screens/Detail/components/DataSection/index.tsx:5]: Sort imports, global imports first!

**Pull request merged**